test/helpers/calculateImageHeight.js

```diff
@@ -0,0 +1,27 @@
+import chai from 'chai';
+import calculateImageHeight from 'helpers/calculateImageHeight';
+
+const { assert } = chai;
+
+describe('calculateImageHeight', () => {
+  it('returns the expected height', () => {
+    const actual = calculateImageHeight(1000, { w: 800, h: 600 });
+    const expected = 750;
+
+    assert.deepEqual(actual, expected);
+  });
+
+  it('returns an integer', () => {
+    const actual = calculateImageHeight(1000, { w: '800', h: '600' });
+    const expected = 750;
+
+    assert.deepEqual(actual, expected);
+  });
+
+  it('fails silently', () => {
+    const actual = calculateImageHeight();
+    const expected = '';
+
+    assert.deepEqual(actual, expected);
```

Comment:
Is this really failing silently? What about `calculateImageHeight(1000, {h: 1000})`. Might return `NaN`.